app/.sentryclirc

```diff
@@ -0,0 +1,2 @@
+[auth]
+token=b3814657fe20451680d5193f67bfa28ab3dd8281882e42c2b2e76ff35c3b1ec6
```

Comment:
This needs to be removed (and an issue opened to update the detect-secrets configuration?)

Reply:
Done thanks.